views/index.erb

```diff
@@ -0,0 +1 @@
+Welcome to Chitter
```

Comment:
can you add links on this page to take you to the other routes? Can simply be:

`<a href="/make_a_peep">Make a peep</a>`
`<a href="/view_peeps">View peeps</a>`

Reply:
Thank you I didn't think about that!